styles/themes/base/base.variables.scss

```diff
@@ -0,0 +1,28 @@
+:root {
+  --main-color: hsl(352, 94%, 40%);
+  --main-color-dark: hsl(352, 94%, 30%);
+  --main-color-light: hsl(352, 94%, 50%);
+  --light-color: white;
```

Comment:
I guess the naming is not generic enough, what is I have a dark theme and want to replace all white to black? In that case "light-color" won't make any sense. Is it more like "base-color" o "background-color"? I would go with something like "base-color" and "base-inverted-color" imo

Reply:
a good question, don't know it myself yet
I'll go with some more components to refine what colors do we need